files/index/index.css

```diff
@@ -1,3 +1,11 @@
 h1 {
     color: #00a3ba;
 }
+
+h2 {
```

Comment:
![image](https://github.com/mrc-ide/wodin-shortcourse-2023/assets/1558093/3e3c299e-3b46-47a1-93c0-b9f1149b0403)

Just putting this here since it's sort of a styling issue - do we need to put something that makes it more obvious that content _will_ arrive at some point? It's not super obvious when you land atm. Thoughts from @lwhittles would be good here

Reply:
Not a huge issue I don't think as things will start to appear by the day they are given this link